app/routes/questions/$question.tsx

```diff
@@ -79,6 +81,7 @@ export function Question({
   useEffect(() => {
     if (text == null && !isLoading.current) {
       isLoading.current = true
+      // This is where the actual contents of the question get fetched from the backend
```

Comment:
I added this comment here, as every couple of weeks I spend 30 minutes trying to find where the questions get loaded :D

Reply:
We should load content of initial questions already with page load, so this is not the only place to load actual contents...

And I think someone changed Coda tables and/or someone is using wrong table id that Coda admins didn't want us to use, and we load all answer text while loading all 700 question titles and then that request is not cache-invalidated somehow, so "it's more complicated".

Reply:
And direct links with question id were supposed to load the contents right away + server-render it too, not to fetch it lazily from useEffect ... that might also need some investigation if it won't be "auto-resolved" by fixing the cache issues 😓